### DISCUSSION TITLE
Welcome to CPP_Optimizations_Diary Discussions!

### DISCUSSION THREAD

**Discussion by facontidavide** [facontidavide/CPP_Optimizations_Diary#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cosmicog** [comment]
This was a requirement after people overused comment feature to show some respect :pray: because of the mails we repository watchers received after the `README.md` update :smiley:

**Reply by facontidavide** [reply]
That was equally hearth-warming and hilarious
